packages/analytics/src/public-types.ts

```diff
@@ -288,4 +288,14 @@ export interface EventParams {
   page_path?: string;
   [key: string]: unknown;
 }
+
+/** Maps the applicable end user consent state. */
+export interface ConsentSettings {
```

Comment:
There are additional options listed in https://developers.google.com/tag-platform/devguides/consent#tracking_consent_state that are not covered in mobile should I add a generic `[key: string]: string` to accommodate or make it match mobile

Reply:
Maybe the best way is to add `[key:string]: string`, so it more or less looks like mobile (as opposed to actually adding the additional types) while not locking out developers adding more if they need.

Reply:
added it as unknown because I see examples in the guide with Arrays, numbers, and strings. Presumably there are quite of few different allowable options